### PR TITLE
Add Group by clause support

### PR DESCRIPTION
**Pull request by marioidival**
refs #153

This PR also add support to Group Functions like SUM, AVG.

### CODE REVIEW

File: adapters/postgres/postgres_test.go [hunk 1]
```diff
@@ -186,6 +186,41 @@ func TestInvalidWhereByRequest(t *testing.T) {
...
+func TestGroupByClause(t *testing.T) {
+	var testCases = []struct {
+		description string
+		url         string
+		expectedSQL string
+		emptyCase   bool
+	}{
+		{"Group by clause with one field", "/prest/public/test5?_groupby=celphone", "GROUP BY celphone", false},
+		{"Group by clause with two fields", "/prest/public/test5?_groupby=celphone,name", "GROUP BY celphone,name", false},
+		{"Group by clause without fields", "/prest/public/test5?_groupby=", "", true},
```
Comment (avelino) [inline adapters/postgres/postgres_test.go]: added test cases:
- field not exist
- special caracter (example: `*`)

File: controllers/tables.go [hunk 2]
```diff
@@ -165,6 +165,14 @@ func SelectFromTables(w http.ResponseWriter, r *http.Request) {
 			requestWhere)
 	}
 
+	// TODO: Add group by support
```
Comment (avelino) [inline controllers/tables.go]: I didn't understand that comment (TODO, codeclimate issue)

File: adapters/postgres/postgres_test.go (hunk 1 above)
Comment (marioidival) [inline adapters/postgres/postgres_test.go]: This two cases will be implemented on `controllers`.

File: controllers/tables.go (hunk 2 above)
Comment (marioidival) [inline controllers/tables.go]: I forgot to remove

**Comment by marioidival** [comment]
WIP yet

**Comment by marioidival** [comment]
Working in README

File: adapters/postgres/postgres.go [hunk 3]
```diff
@@ -661,3 +677,31 @@ func ColumnsByRequest(r *http.Request) []string {
...
+
+// GroupByClause get params in request to add group by clause
+func GroupByClause(r *http.Request) (groupBySQL string) {
+	queries := r.URL.Query()
+	countQuery := queries.Get("_groupby")
+	if countQuery == "" {
+		return
+	}
+
+	groupBySQL = fmt.Sprintf("GROUP BY %s", countQuery)
```
Comment (felipeweb) [inline adapters/postgres/postgres.go]: move this to statements

File: adapters/postgres/postgres.go [hunk 4]
```diff
@@ -661,3 +677,31 @@ func ColumnsByRequest(r *http.Request) []string {
 	}
 	return columns
 }
+
+// GroupByClause get params in request to add group by clause
+func GroupByClause(r *http.Request) (groupBySQL string) {
+	queries := r.URL.Query()
+	countQuery := queries.Get("_groupby")
```
Comment (felipeweb) [inline adapters/postgres/postgres.go]: countQuery???

File: adapters/postgres/postgres.go (hunk 3 above)
Comment (marioidival) [inline adapters/postgres/postgres.go]: done

**Pull request merged**